instrument/D17_Definition.xml

```diff
@@ -91,7 +93,7 @@
   <type name="slit3"/>
   <!--DETECTORS-->
   <!--64 TUBES FORM ONE DETECTOR-->
-  <component idfillbyfirst="x" idstart="1" idstepbyrow="1" type="detector">
+  <component idfillbyfirst="x" idstart="0" idstepbyrow="1" type="detector">
```

Comment:
This is a change in the generator script, right?

Reply:
Yes, I added a unit test in `LoadILLReflectometry` to check that detector IDs as well as spectrum numbers start with 0. So, we're safe.

For the generator script, I did not yet push this change but will do and reference it here.